velox/dwio/parquet/tests/reader/E2EFilterTest.cpp

```diff
@@ -336,7 +336,7 @@ TEST_F(E2EFilterTest, shortDecimalDictionary) {
               true);
         },
         false,
-        {"shortdecimal_val"},
+        {},
```

Comment:
Let's add the check to forbid filter and hook on parquet decimal column reader in the PR as well

Reply:
@Yuhta Already added.